### DISCUSSION TITLE
Aliasing `aura` to `aura -Syu`

### DISCUSSION THREAD

**Discussion by xfzv** [fosskers/aura#878] [Ideas]
Currently, running `aura` prints the help menu (like `aura -h` or `aura --help`), which isn't very useful.

Some AUR helpers like `paru` or `yay` have aliased the program command to `$program -Syu`

Sources:

- [paru](https://github.com/Morganamilo/paru/blob/master/README.md?plain=1#L48)
- [yay](https://github.com/Jguer/yay/blob/next/README.md?plain=1#L99)

I think aliasing `aura` to `aura -Syu` instead of displaying the help menu would be a better default.

Thoughts?

**Comment by fosskers** [comment]
In my experience it's actually quite common for programs to default to "help" if given no other options. Thank you for the suggestion, but in this case I will be maintaining the current behaviour.

**Reply by xfzv** [reply]
OK, thanks for the reply!

**Comment by yvvki** [comment]
It's easy to make a wrapper to `aura` (a wrapper for a wrapper?), just check whether you have any flags and off to do some updates.

```bash
#!/bin/bash

if [[ $# -eq 0 ]]; then
  /usr/bin/aura -Syu && /usr/bin/aura -Ayu
else
  /usr/bin/aura $@
fi

```

**Reply by fosskers** [reply]
Aura occasionally calls to `pacman` for certain things, but most of its code is independent, so while it's common to call such tools "wrappers", it isn't strictly correct.

**Reply by yvvki** [reply]
I mean, a wrapper wrapper sounds cooler than a wrapper to isn't strictly wrapper 😅
